wgpu-core/src/device/mod.rs

```diff
@@ -1004,7 +1004,8 @@ impl<F: IdentityFilter<id::BindGroupId>> Global<F> {
         let bind_group_layout = &bind_group_layout_guard[desc.layout];
         let bindings =
             unsafe { slice::from_raw_parts(desc.bindings, desc.bindings_length as usize) };
-        assert_eq!(bindings.len(), bind_group_layout.bindings.len());
+        assert_eq!(bindings.len(), bind_group_layout.bindings.len(), format!("The bind group {:?} has {} bindings 
```

Comment:
you don't need `format!` when inside the `assert!`

Reply:
OK no problem, I'll update and send.

Reply:
done